src/analysis/special_functions/pow.lean

```diff
@@ -1248,6 +1255,44 @@ lemma rpow_le_rpow_of_exponent_ge {x : ℝ≥0} {y z : ℝ} (hx0 : 0 < x) (hx1 :
   x^y ≤ x^z :=
 real.rpow_le_rpow_of_exponent_ge hx0 hx1 hyz
 
+lemma le_neg_rpow_iff {a b c : ℝ} (ha : 0 < a) (hb : 0 < b) (hc : c < 0) :
```

Comment:
```suggestion
lemma le_rpow_iff_of_neg {a b c : ℝ} (ha : 0 < a) (hb : 0 < b) (hc : c < 0) :
```

Reply:
(and same for the other three)

Reply:
I've changed the even further to include the inversion that is now in the lhs of the iff. If that is not correct, I can revert that of course.